package.json

```diff
@@ -147,6 +148,7 @@
     "svelte-adapter": "^0.3.0",
     "svelte-loader": "^2.13.6",
     "svelte-preprocess": "2.12.0",
+    "turndown": "^7.1.1",
```

Comment:
`turndown` dependency is already present in the `master`'s [package.json](https://github.com/santiment/app/blob/master/package.json#L102)